package.json

```diff
@@ -40,12 +40,12 @@
   },
   "homepage": "https://github.com/webgme/user-management-page#readme",
   "dependencies": {
-    "body-parser": "1.15.2",
+    "body-parser": "^1.18.3",
     "ejs": "^2.4.2",
     "express": "4.16.3"
   },
   "devDependencies": {
-    "admin-lte": "2.4.2",
+    "admin-lte": "^2.4.3",
```

Comment:
I'd be careful with this one.. They have introduced breaking changes before.. Bump it - but tie the version.